src/base/TilesRendererBase.js

```diff
@@ -14,12 +14,17 @@ import { UNLOADED, LOADING, PARSING, LOADED, FAILED } from './constants.js';
  */
 const priorityCallback = ( a, b ) => {
 
-	if ( a.__lastFrameVisited !== b.__lastFrameVisited ) {
+	if ( a.__inFrustum !== b.__inFrustum ) {
```

Comment:
In the Tile interface in TilesRendererBase.d.ts: add __inFrustum, __used, and remove __lastFrameVisited